tests/system/python/api/test_south_service_tuning.py

```diff
@@ -0,0 +1,389 @@
+            "readingsPerSec": "30",        # 2 seconds interval
+            "maxSendLatency": "10000",     # 10 seconds
+            "bufferThreshold": "20"        # Buffer 20 readings
+        }
+        put_url = f"/fledge/category/{service_name}Advanced"
+        print(f"Configuring advanced parameters for {service_name}: {advanced_config}", put_url)
+        resp = utils.put_request(fledge_url, urllib.parse.quote(put_url), advanced_config)
+        
+        try:
+            # Test matrix of different configurations
+            test_configs = [
+                {"bufferThreshold": "50", "maxSendLatency": "15000", "expected_behavior": "delayed_sends"},
+                {"bufferThreshold": "5", "maxSendLatency": "5000", "expected_behavior": "frequent_sends"},
+                {"bufferThreshold": "100", "maxSendLatency": "2000", "expected_behavior": "latency_driven"},
+                {"bufferThreshold": "10", "maxSendLatency": "30000", "expected_behavior": "threshold_driven"}
+            ]
+            
+            for i, test_config in enumerate(test_configs):
+                print(f"\n--- Test Configuration {i+1}: {test_config['expected_behavior']} ---")
+                
+                # Get baseline
+                baseline_ping = utils.get_request(fledge_url, "/fledge/ping")
+                baseline_count = baseline_ping["dataRead"]
+                
+                # Update configuration
+                put_url = f"/fledge/category/{service_name}Advanced"
+                config_update = {
+                    "bufferThreshold": test_config["bufferThreshold"],
+                    "maxSendLatency": test_config["maxSendLatency"]
+                }
+                resp = utils.put_request(fledge_url, urllib.parse.quote(put_url), config_update)
+                
+                # Verify configuration was applied
+                assert test_config["bufferThreshold"] == resp["bufferThreshold"]["value"]
+                assert test_config["maxSendLatency"] == resp["maxSendLatency"]["value"]
+                
+                # Wait and measure results
+                time.sleep(wait_time * 8)  # 40 seconds
+                
+                new_ping = utils.get_request(fledge_url, "/fledge/ping")
+                new_count = new_ping["dataRead"]
+                readings_collected = new_count - baseline_count
+                
+                print(f"Configuration: bufferThreshold={test_config['bufferThreshold']}, "
+                      f"maxSendLatency={test_config['maxSendLatency']}")
+                print(f"Readings collected in 40s: {readings_collected}")
+                
+                # Basic validation - should always collect some readings
+                assert readings_collected > 0, "Should have collected some readings in 40 seconds"
+                
+                # Store results for comparison
+                test_config["actual_readings"] = readings_collected
```

Comment:
You're comparing only dataRead, which tracks how much the plugin generates.
But your configs (bufferThreshold, maxSendLatency) control when readings are sent to storage, not generation.

Then compare actual_sent to understand send behavior?